_maps/map_files/Minerva/TGS_Minerva.dmm

```diff
@@ -6174,6 +6170,7 @@
 /obj/structure/cable,
 /obj/machinery/atmospherics/pipe/simple/supply/hidden/layer1,
 /obj/machinery/atmospherics/pipe/simple/scrubbers/hidden,
+/mob/living/simple_animal/corgi/Ranger,
```

Comment:
that is not a corgi

Reply:
Pug is also put as corgi, which is the reason why I put Ranger under corgi.